ext/curl/interface.c

```diff
@@ -1547,6 +1547,8 @@ static size_t curl_read(char *data, size_t size, size_t nmemb, void *ctx)
 				if (Z_TYPE(retval) == IS_STRING) {
 					length = MIN((int) (size * nmemb), Z_STRLEN(retval));
 					memcpy(data, Z_STRVAL(retval), length);
+				} else if (Z_TYPE(retval) == IS_LONG) {
+					length = zval_get_long(&retval);
```

Comment:
Shouldn't we expose the `CURL_READFUNC_PAUSE` and the `CURL_READFUNC_ABORT` constants too?

Also, if the zval is known to be a long then feel free to simply use `Z_LVAL_P()`

Reply:
@kocsismate Thanks. You're right I fixed it to use `Z_LVAL_P()`. The `CURL_READFUNC_PAUSE` constant is already exposed. 

I could expose `CURL_READFUNC_ABORT` but I would do only on the master branch. I consider this PR a bugfix since the constant is there but not usable, but all other constants should be added on master. What do you think ?

Reply:
> The CURL_READFUNC_PAUSE constant is already exposed.

Oh nice! Somehow my search didn't find it.. I'm not sure what went wrong, but whatever :)

> I could expose CURL_READFUNC_ABORT but I would do only on the master branch. I consider this PR a bugfix since the constant is there but not usable, but all other constants should be added on master. What do you think ?

Yes, master seems reasonable :)